src/Model/Cart.java

```diff
@@ -0,0 +1,81 @@
+package Model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import API.CartService;
+import Model.MailingList.EmailSender;
+import Model.paymentMethods.FinancialPayment;
+
+
+/**
+ * @author Jordan
+ * shopping cart
+ */
+public class Cart implements CartService{
```

Comment:
This class name and implementation is wrong. A good name for this class could be CartServiceImpl. CartServiceImpl must handle or manage a Cart.